cmd/peer-rest-client.go

```diff
@@ -408,133 +408,162 @@ func (client *peerRESTClient) GetAllBucketStats() (BucketStatsMap, error) {
 
 // LoadBucketMetadata - load bucket metadata
 func (client *peerRESTClient) LoadBucketMetadata(bucket string) error {
-	values := make(url.Values)
-	values.Set(peerRESTBucket, bucket)
-	respBody, err := client.call(peerRESTMethodLoadBucketMetadata, values, nil, -1)
+	conn := client.gridConn()
+	if conn == nil {
+		return nil
+	}
+	_, err := loadBucketMetadataHandler.Call(context.Background(), conn, grid.NewMSSWith(map[string]string{
+		peerRESTBucket: bucket,
+	}))
 	if err != nil {
```

Comment:
Instead of 
```
if err != nil {
    return err
}
return nil
```

we can simply do this:
```
return err
```

(and same for all the other functions)

Reply:
I will get to these subsequently @fwessels. Thanks in another PR.